app/assets/stylesheets/application.css

```diff
@@ -121,3 +121,11 @@
   margin: 10px;
   margin-bottom: 20px;
 }
+
+.login .loginn {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
```

Comment:
- [ ] Dear @Ouail-01, The views in the device for the registration and login is expected to be modified to match the wireframes and their styling. Please update your styling to match the wireframe. 

### Your login page 
![image](https://github.com/Ouail-01/Blog-App/assets/69077061/5b8e2425-bb1b-4997-a732-46f611da7499)
### The template 
![blog_login](https://github.com/Ouail-01/Blog-App/assets/69077061/1e91e9ea-157d-4e7c-a061-370850125eae)